modular_zubbers/code/modules/clothing/outfits/outlast.dm

```diff
@@ -20,20 +20,35 @@
 		/obj/item/surgical_drapes,
 		/obj/item/storage/pill_bottle,
 		/obj/item/tank/internals/emergency_oxygen,
+		/obj/item/forging/hammer,
+		/obj/item/forging/tongs,
+		/obj/item/forging/billow,
 	)
-/obj/item/clothing/glasses/surgerygoggles //Sprite by pacertest on discord
-	name = "Surgery Goggles"
+
+/obj/item/clothing/glasses/hud/health/surgerygoggles //Sprite by pacertest on discord
+	name = "surgery goggles"
 	desc = "A pair of goggles worn to protect the eyes from viscera flying off a patient, a wonderful development in medical technology."
 	icon = 'modular_zubbers/icons/obj/clothing/outlast.dmi'
 	worn_icon = 'modular_zubbers/icons/mob/clothing/outlast.dmi'
 	icon_state = "surgerygoggles"
-/obj/item/clothing/gloves/longrubbergloves //Sprite by pacertest on discord
-	name = "Long Rubber Gloves"
+
+/obj/item/clothing/glasses/surgerygoggles //Sprite by pacertest on discord
+	name = "recovery goggles"
+	desc = "Like biker goggles, but creepier!"
+	icon = 'modular_zubbers/icons/obj/clothing/outlast.dmi'
+	worn_icon = 'modular_zubbers/icons/mob/clothing/outlast.dmi'
+	icon_state = "surgerygoggles"
+
+/obj/item/clothing/gloves/latex/nitrile/longrubbergloves //Sprite by pacertest on discord
+	name = "long rubber medical gloves"
 	desc = "Pricy somewhat sterile gloves that are thicker than latex, these ones extend almost past the elbow. Excellent grip ensures very fast carrying of patients along with the faster use time of various chemical related items."
 	icon = 'modular_zubbers/icons/obj/clothing/outlast.dmi'
 	worn_icon = 'modular_zubbers/icons/mob/clothing/outlast.dmi'
 	icon_state = "longrubber"
-	clothing_traits = list(
-		TRAIT_QUICKER_CARRY,
-		TRAIT_FASTMED
-	)
```

Comment:
Just for anyone that happens to stumble upon this: These aren't removed, they are still inherited from the parent /obj/item/clothing/gloves/latex/nitrile

Reply:
Yup. That's why I moved it up. Cleaner.